File: src/main/java/com/example/entities/ClosureReason.java
package com.example.entities;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "closure_reason")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class ClosureReason {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "closure_reason_id")
    private int closureReasonId;

    @Column(name = "closure_reason_desc")
    private String closureReasonDesc;

    @Column(name = "enquirer_name")
    private String enquirerName;

    public int getClosureReasonId() {
		return closureReasonId;
	}

	public void setClosureReasonId(int closureReasonId) {
		this.closureReasonId = closureReasonId;
	}

	public String getClosureReasonDesc() {
		return closureReasonDesc;
	}

	public void setClosureReasonDesc(String closureReasonDesc) {
		this.closureReasonDesc = closureReasonDesc;
	}

	public String getEnquirerName() {
		return enquirerName;
	}

	public void setEnquirerName(String enquirerName) {
		this.enquirerName = enquirerName;
	}

	public ClosureReason(String closureReasonDesc,String enquirerName){
        this.closureReasonDesc =closureReasonDesc;
        this.enquirerName = enquirerName;
    }
}

File: ComputerSeekho.Net/ComputerSeekho.Net/Models/ClosureReason.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ComputerSeekho.Net.Models;

[Table("closure_reason")]
public class ClosureReason
{
    [Key]
    [Column("closure_reason_id")]
    public int ClosureReasonId { get; set; }

    [Column("closure_reason_desc")]
    public string? ClosureReasonDesc { get; set; }

    [Column("enquirer_name")]
    public string? EnquirerName { get; set; }
}
